internal/services/models.go

```diff
@@ -152,8 +152,9 @@ type UserDeviceAPIIntegrationsMetadata struct {
 }
 
 type UserDeviceAPIIntegrationsMetadataCommands struct {
-	Enabled []string `json:"enabled,omitempty"`
-	Capable []string `json:"capable,omitempty"`
+	Enabled  []string `json:"enabled,omitempty"`
+	Capable  []string `json:"capable,omitempty"`
+	Disabled []string `json:"disabled,omitempty"`
```

Comment:
Do we need a new `Disabled` field since we already have `Capable`? If there is a Command in `Capable` but not in `Enabled` this should imply the command is disabled.

Reply:
This makes sense. However, `Capable` is supposed to hold all available commands, splitting things out into a new field called `Disabled` helps avoid having to compute each time what is disabled/enabled which makes the commands easier to work with as we scale or increase the number of available commands.

Reply:
I don’t think we ever ended up using the enabled/disabled stuff—the frontend stores the preference.